flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/JobManagerTable.java

```diff
@@ -61,4 +73,20 @@
 	 */
 	@Nullable
 	JobManagerConnection get(JobID jobId);
+
+	/**
+	 * Gets the {@link JobManagerConnection} stored under the given {@link ResourceID}.
+	 *
+	 * @param resourceId resourceId identifying the {@link JobManagerConnection} to return
+	 * @return the {@link JobManagerConnection} stored under resourceId; {@code null} otherwise
+	 */
+	@Nullable
+	JobManagerConnection get(ResourceID resourceId);
```

Comment:
maybe return `Optional` instead

Reply:
This class has been removed in later commits.